packages/core/ios/Sources/JsRefreshRate.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+#import <Foundation/Foundation.h>
+
+@interface JsRefreshRate : NSObject<RCTBridgeModule>
```

Comment:
/nit prefix are uppercased in objc
```suggestion
@interface JSRefreshRate : NSObject<RCTBridgeModule>
```